src/utils_rmw.c

```diff
@@ -143,3 +143,32 @@ human_readable_size (off_t size)
 
   return buffer;
 }
+
+/*!
+ * Verify with the user about an action. The prompt produced will be
```

Comment:
I'd omit the prompt text from the function description, to reduce the need to update both later.

A developer can clearly see what text is. for a simple function like this, you might just say, "prompt user for confirmation". Then for the '@return` .. "@returns true if 'y' was entered, false otherwise"

I appreciate your initiative to document your new functions! :)